src/lib/aiEnhancement.ts

```
@@ -0,0 +1,247 @@
+          border: "1px solid #7d3650",
+        },
+        duration: 5000, // Show error for longer
+      });
+
+      throw error;
+    }
+  } catch (error) {
+    // Re-throw the error to be handled by the caller
+    throw error;
+  }
+}
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)